app/components/home/hero/Hero.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+
+import { Button } from "@components/button";
+import { TRANSLATION } from "./hero.constants";
```

Comment:
These constants only belong to the hero component. if in case needed we have the index.ts file in that component. Also if we start to put all the local constants files of every component in the future it will be really problematic to manage. So I think we should only put those constant files in the constant folder which will be shared by others or at least have good possibilities to be shared.